simpleagent/simpleagent.py

```diff
@@ -12,52 +16,91 @@ def __init__(self, instructions, client: SendToLLM, number_of_turns = 5):
             self.start_project(number_of_turns)
 
     def generate_random_agent_name(self):
-        firstname = agent_firstnames[randint(0,len(agent_firstnames)-1)]
-        lastname = agent_lastnames[randint(0,len(agent_lastnames)-1)]
+        firstname = agent_firstnames[randint(0, len(agent_firstnames) - 1)]
+        lastname = agent_lastnames[randint(0, len(agent_lastnames) - 1)]
         return f"{firstname} {lastname}"
 
-
     def generate_prompt_from_instructions(self, instructions):
-        self.prompt_generation_prefix = """You are an administrator.  A user has given instructions for a task that must be completed.

```

Comment:
Hi Jonathan.  This is all excellent work.  You solved a few problems I didn't know I had.  However, this is the only part that I would change.  You are correct, all of this could and should be packaged up into a variable.  However, I want to make the code as simple as possible for anyone to read.  I want people to be able to just scroll through this file and know what all the prompts say and where, so I would rather not store this prompt in another file.  Definitely a good idea for efficiency, but for right now what we are looking for is readability.